playground/hoa0217/java-springboot-order-perf-multi-module/catalog/src/main/resources/application.yml

```
@@ -0,0 +1,23 @@
+      ddl-auto: none
+    properties:
+      hibernate:
+        format_sql: true
+        show_sql: true
+  h2:
+    console:
+      enabled: true
+      path: '/h2-console'
+
+server:
+  port: 8082
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)